### PR TITLE
Grub efi amd64 image

### PR DESCRIPTION
**Pull request by tomirgang**
# Changes

- Add a amd64 example image which makes use of grub-efi.

# Dependencies:

none

# Tests results

```bash
<!-- add Robot test CLI logs here. -->
```

# Developer Checklist:

- [x] Test: Changes are tested
- [ ] Doc: Documentation has been updated 
- [x] Git: Informative git commit message(s)
- N/A Issue: If a related GitHub issue exists, linking is done

## Checklists for documentation

- [ ] Grammar: the content is grammatically correct
      (spelling, grammar, formatting, US English is used)
- [ ] Comprehensibility/Unambiguousness: the content is easy readable, clear to understand
- [ ] Correctness and consistency: the content is technically correct and consistent,
      no contradictions, no double descriptions
- [ ] Terminology: technical terms are clear and they are used correctly and documented in the glossary
- [ ] Level of detail: the content matches the detail level necessary for the reviewed object

# Reviewer checklist:

- [ ] Review: Changes are reviewed
- [ ] Review: Tested by the reviewer

## Checklists for documentation

- [ ] Grammar: the content is grammatically correct
      (spelling, grammar, formatting, US English is used)
- [ ] Comprehensibility/Unambiguousness: the content is easy readable, clear to understand
- [ ] Correctness and consistency: the content is technically correct and consistent,
      no contradictions, no double descriptions
- [ ] Terminology: technical terms are clear and they are used correctly and documented in the glossary
- [ ] Level of detail: the content matches the detail level necessary for the reviewed object

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
 {
 	"name": "EB corbos Linux SDK 1.4.0",
-	"image": "ghcr.io/elektrobit/ebcl_dev_container:v1.4.1",
+	"image": "ghcr.io/elektrobit/ebcl_dev_container:v1.4.2",
```
Comment (simone-weiss) [inline .devcontainer/devcontainer.json:5]: Also lets talk on the branching for 2.x lines in other repos

File: images/amd64/qemu/grub-efi/crinit/Taskfile.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+tasks:
+  default:
+    cmds:
+      - task: InitrdGenerator:build-initrd-img
+      - task: RootGenerator:build-root-filesystem-tarball
+      - task: RootGenerator:config-root-filesystem-tarball
+      - task: Embdgen:build-image
+      - task: Efi:install-grub
+      - task: Qemu:run-qemu-efi-amd64
```
Comment (simone-weiss) [inline images/amd64/qemu/grub-efi/crinit/Taskfile.yml:37]: add method none pls

File: images/amd64/qemu/grub-efi/xen/Taskfile.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+tasks:
+  default:
+    cmds:
+      - task: InitrdGenerator:build-initrd-img
+      - task: RootGenerator:build-root-filesystem-tarball
+      - task: RootGenerator:config-root-filesystem-tarball
+      - task: Embdgen:build-image
+      - task: Efi:install-grub
+      - task: Qemu:run-qemu-efi-amd64
```
Comment (simone-weiss) [inline images/amd64/qemu/grub-efi/xen/Taskfile.yml:37]: same

**Comment by thir820** [comment]
Skipping the documentation for the new image now. Images needs to be aligned and clean up.

**Pull request merged**